pdns/recursordist/docs/settings.rst

```diff
@@ -1052,11 +1055,14 @@ returning back to normal processing and handling other events.
 
 ``minimum-ttl-override``
 ------------------------
+.. versionchanged:: 4.5.0
+  Old versions used default 0.
+
 -  Integer
--  Default: 0 (disabled)
+-  Default: 1
 
 This setting artificially raises all TTLs to be at least this long.
-While this is a gross hack, and violates RFCs, under conditions of DoS, it may enable you to continue serving your customers.
+Setting this to a value greater than 1 technically is an RFC violation, but might improve performance a lot.
 Can be set at runtime using ``rec_control set-minimum-ttl 3600``.
```

Comment:
It would be good to have a sentence or two on why setting this to `0` might be a bad idea